server.js

```diff
@@ -17,32 +17,49 @@ if (!settings || !settings.apiToken) {
 
 const client = new postmark.ServerClient(settings.apiToken);
 
-export const sendEmail = async ({ to, subject, content, from: fromParam }) => {
-  const from = fromParam || settings.from;
+export const sendEmail = async (options) => {
```

Comment:
a function call without any arg would break in the next line.

it would be better to have async (options = {}) to avoid undefined errors destructing args.